### ISSUE TITLE
Azure DevOps sass processing error

### ISSUE THREAD

**Issue by drmikesell** [beingnin/Azure-Devops-Extension-Sass-Compiler#4]
I am having problems successfully using this extension on Azure devops.  Here is a snip of the pipeline:
```
pool:
  vmImage: "ubuntu-latest"

- task: UseDotNet@2
  displayName: "Use .NET Core SDK 6"
  inputs:
    packageType: "sdk"
    version: "6.x"

- task: DotNetCoreCLI@2
  displayName: "dotnet restore"
  inputs:
    command: restore
    feedsToUse: config
    nugetConfigPath: nuget.config
    projects: "Atlas.CompositionEngine.sln"

- task: NodeTool@0
  inputs:
    versionSource: 'spec'
    versionSpec: '6.x'
 
- task: process-sass-files@0
  displayName: 'Compile sass'
  inputs:
    inputFile: '$(Build.SourcesDirectory)/wwwroot/css/inspinia/style.scss'
    outputFile: '$(Build.SourcesDirectory)/wwwroot/css/inspinia/style.css'
    style: 'compressed'
    enableVendorPrefixing: true
    sassVersion: '1.39.x'
```

When the process-sass-files task executes, autoprefixer-cli installs, then the following error is displayed:
```
sass compilation thrown error
{ Error: spawn /bin/sh ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn /bin/sh',
  path: '/bin/sh',
  spawnargs:
   [ '-c',
     'sass "D:\\Sample\\sample.scss" "D:\\Sample\\sample.css" --style compressed --no-source-map' ] }
##[error]Error: spawn /bin/sh ENOENT
```

What piece of the puzzle am I missing?  Any assistance is appreciated.

**Comment by Fandermill** [comment]
I'm having the same issue. Is seems that it is looking for "D:\Sample\sample.scss", but that file does not exists in my pipeline.
Looking at the code in this repo, I find this in `index.js`:

```js
//tests: remove later
inputFile = 'D:\\Sample\\sample.scss';
outputFile = 'D:\\Sample\\sample.css';
```

So, I guess @beingnin forgot to remove his testing data before releasing a new version ;)

**Comment by beingnin** [comment]
@Fandermill You were right. It was caused by a wrong package and the issue is fixed on 0.2.2 version of the extension. Please use it and let me know if any issue persist. @drmikesell

**Comment by beingnin** [comment]
I hope the issue would be resolved with the new version
